README.md

```diff
@@ -30,3 +30,11 @@ To run tests in your docker container, you will need to override the environment
 ```
 docker-compose exec -e RAILS_ENV=test app rails test
 ```
+
+## Getting Started
+
+Once your server is running you can hit the api! 
+ex. 'api/v3/public/cards/sure_gamble'
+
+hitting any invalid route shows a full list of routes
+ex. 'api/v3/public/'
```

Comment:
Actually, there is a URL for this: http://localhost:3000/rails/info/routes

Reply:
I knew there was some secret I hadn't discovered!